### PR TITLE
Add sp attributes in DataSource

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
- Add missing sp attributes in dataSource file
- Aligned all the schema attributes in alphabetical order
- updated READ method

### Check List
- [x] New functionality includes testing.
  - [x]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by yuvirani — APPROVED** [review]

File: oneview/data_source_server_profile.go [hunk 1]
```diff
@@ -59,6 +59,10 @@ func dataSourceServerProfile() *schema.Resource {
 							Type:     schema.TypeString,
 							Required: true,
 						},
```
Comment (nabhajit-ray) [inline oneview/data_source_server_profile.go:61]: associated server is missing

File: oneview/data_source_server_profile.go [hunk 2]
```diff
@@ -742,6 +742,119 @@ func dataSourceServerProfile() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
```
Comment (nabhajit-ray) [inline oneview/data_source_server_profile.go:743]: boot.target is missing

File: oneview/data_source_server_profile.go [hunk 3]
```diff
@@ -742,6 +742,119 @@ func dataSourceServerProfile() *schema.Resource {
...
 			},
+			"category": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created": {
```
Comment (nabhajit-ray) [inline oneview/data_source_server_profile.go:749]: requestedMbps iis mssing and have to fixed in connection settings

File: oneview/data_source_server_profile.go [hunk 4]
```diff
@@ -742,6 +742,119 @@ func dataSourceServerProfile() *schema.Resource {
...
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"etag": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```
Comment (nabhajit-ray) [inline oneview/data_source_server_profile.go:760]: enclosure uri, initial scope uri  missing.

File: oneview/data_source_server_profile.go (hunk 1 above)
Comment (VenkateshRavula) [reply oneview/data_source_server_profile.go:61]: added

File: oneview/data_source_server_profile.go (hunk 3 above)
Comment (VenkateshRavula) [reply oneview/data_source_server_profile.go:749]: removed that line.

File: oneview/data_source_server_profile.go (hunk 2 above)
Comment (VenkateshRavula) [reply oneview/data_source_server_profile.go:743]: added

File: oneview/data_source_server_profile.go (hunk 4 above)
Comment (VenkateshRavula) [reply oneview/data_source_server_profile.go:760]: added ```enclosure_uri``` and ```initial_scope_uris```

**Review by nabhajit-ray — APPROVED** [review]

**Pull request merged**